packages/expo-bluetooth-utils/src/Bluetooth.ts

```diff
@@ -0,0 +1,34 @@
+import { Base64 } from 'js-base64';
+
+export { default as Characteristics } from './Characteristics';
+export { default as Descriptors } from './Descriptors';
+export { default as Services } from './Services';
+
+/*
+ * Convert a JS value before sending it to native.
+ * 1. Use encodeURIComponent to get percent-encoded UTF-8 string value.
+ * 2. Convert the percent-encodings to raw bytes.
+ * 3. Pass value to Base64.btoa (Binary to ASCII)
+ */
+export function JSONToNative(str: string): string {
```

Comment:
The "JSON" part here is confusing - there's no JSON at all -- name it something like `encodeBinaryString`